pom.xml

```diff
@@ -35,6 +35,7 @@
   </scm>
 
   <properties>
+    <vertx.version>4.0.0</vertx.version>
```

Comment:
Shouldn't this be 4.1.2?

Reply:
I had been working with 4.0 but am fine with going to the latest.  I have made this change and re-checked it in (squashed).